### DISCUSSION TITLE
Daily note breadcrumb workflow

### DISCUSSION THREAD

**Discussion by JonathanBuchh** [BryanJenksCommunity/FAQ#37] [Q&A] [answered]
Hi Bryan,

You mentioned in a comment on one of your videos that you use an Alfred snippet to create breadcrumbs in your daily notes and I was wondering if you could go more in-depth (prefix, postfix, keyword, snippet itself). I was also wondering if you can only use the snippet on the daily note that is the current day and not a past/future daily note (I assume it does not, but maybe you have a workaround).

Thanks!
Jonathan

**Comment by tallguyjenks** [comment, marked as answer]
Yes! the alfred snippet in its entirity is this:

`[[{date -1d:YYYY-MM-dd}]] <== <button class="date_button_today">Today</button> ==> [[{date +1d:YYYY-MM-dd}]]`

you may also be able to imitate this behavior with the templater plugin :)
